core/src/main/java/com/linecorp/armeria/server/ServiceRequestContextBuilder.java

```diff
@@ -62,6 +65,26 @@ public void serverStarting(Server server) throws Exception {
         }
     };
 
+    private static final TimeoutTask noopTimeoutTask = new TimeoutTask() {
+        @Override
+        public boolean canSchedule() {
+            return true;
+        }
+
+        @Override
+        public void run() { /* no-op */ }
+    };
+
+    /**
+     * A timeout controller that has been timed-out.
+     */
+    private static final DefaultTimeoutController noopTimedOutController =
+            new DefaultTimeoutController(noopTimeoutTask, CommonPools.workerGroup().next());
```

Comment:
Can we use `ImmediateEventExecutor`, so that we do not instantiate event loop threads in `CommonPools` unnecessarily during the class initialization?